projects/addon-table/components/table/table-expand/table-expand.component.ts

```diff
@@ -0,0 +1,76 @@
+import {isPlatformServer} from '@angular/common';
+import type {ElementRef} from '@angular/core';
+import {
+    ChangeDetectionStrategy,
+    Component,
+    computed,
+    EventEmitter,
+    inject,
+    Input,
+    Output,
+    PLATFORM_ID,
+    signal,
+    ViewChild,
+} from '@angular/core';
+import {tuiInjectElement} from '@taiga-ui/cdk/utils/dom';
+
+import {TUI_TABLE_OPTIONS} from '../table.options';
+
+@Component({
+    standalone: true,
+    selector: 'tui-table-expand',
+    templateUrl: './table-expand.template.html',
+    styleUrls: ['./table-expand.style.less'],
+    changeDetection: ChangeDetectionStrategy.OnPush,
+    host: {
+        style: '--tui-duration: 0ms',
+    },
+})
+export class TuiTableExpand {
```

Comment:
Looks like we need to disabled and re-enable animations everytime the component leaves and enters DOM, check out tuiPresent directive

https://github.com/user-attachments/assets/c3c54d9a-2b44-4b7b-8b33-e9a640ba5913